packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -225,6 +257,34 @@ contract Attestations is
     );
   }
 
+  /**
+   * @notice Reveals the issuers for the most recent attestation request.
+   * @param identifier The hash of the identifier to be attested.
+   */
+  function revealIssuers(bytes32 identifier) external {
+    IdentifierState storage state = identifiers[identifier];
+
+    require(
+      state.requests[msg.sender].blockNumber != 0,
+      "No active attestation request to reveal issuers for"
```

Comment:
#esl, not sure what a better alternative would be. `current`?

Reply:
`expired`/`unexpired`?

Reply:
Yeah I guess `expired` isn't right, since we use that to mean something else elsewhere. `unselected`? I'm not sure, but I don't think `active` is a good choice